client/components/wcc-settings-form/index.js

```diff
@@ -79,10 +80,11 @@ const getFormErrors = ( schema, data ) => {
 		 * This removes the `data.` prepending all errors, to facilitate easier matching to form fields.
 		 */
 		return validate.errors.map( ( error ) => {
-			if ( 0 === error.field.indexOf( 'data.' ) ) {
-				return error.field.substr( 5 );
+			const errorPath = ObjectPath.parse( error.field );
```

Comment:
This PR solves a problem with the usage of `.split( '.' )` - the service IDs for Canada Post contain `.` characters.